src/main/resources/database/changes/release-10.50.0/update_action_plan_add_social.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO action.actiontype
+VALUES
+(4, 'SOCIALNOT', 'Enrolment Invitation Letter', 'Printer', 'n', 'n'),
```

Comment:
The primary keys will now need changing to 8, 9 and 10.
Can we also add `ON CONFLICT DO NOTHING;` to avoid the application falling over.
And maybe be more explicit here? `INTO action.actiontype(actiontypepk, name, description, handler, cancancel, responserequired)`

Reply:
I think it should fall over on conflict to indicate the data is being set up wrong?

Reply:
Depends if these have been fed in manually in prod or not? Other action plans were already in prod and the scripts are being added in after. I guess these are for social so they shouldn't be in there already...